kafka-impl/src/main/java/io/streamnative/pulsar/handlers/kop/KafkaProtocolHandler.java

```diff
@@ -328,7 +328,7 @@ public void initGroupCoordinator(BrokerService service) throws Exception {
 
     public void startGroupCoordinator() throws Exception {
         if (this.groupCoordinator != null) {
-            this.groupCoordinator.startup(false);
+            this.groupCoordinator.startup(true);
```

Comment:
Should we make it a configurable parameter?

Reply:
The `startup` method only start a regular group metadata cache cleanup work whose interval is determined by offset config `offsetsRetentionCheckIntervalMs`, see https://github.com/streamnative/kop/blob/d684f4bd295ed686aa6c6f371864f29f91c02e00/kafka-impl/src/main/java/io/streamnative/pulsar/handlers/kop/coordinator/group/GroupMetadataManager.java#L239-L247

The `false` option is only used in unit test to disable the cleanup work. By the way, the group coordinator was ported from Kafka's code, it's just a mistake before.